common/src/main/java/com/microsoft/identity/common/internal/commands/InteractiveTokenCommand.java

```diff
@@ -39,21 +43,53 @@
 @EqualsAndHashCode(callSuper = true)
 public class InteractiveTokenCommand extends TokenCommand {
     private static final String TAG = InteractiveTokenCommand.class.getSimpleName();
+    //Normally all tasks have an affinity unless configured explicitly for multi-window support to not have one
+    private boolean mHasTaskAffinity = true;
+    private int mTaskId = 0;
 
     public InteractiveTokenCommand(@NonNull final InteractiveTokenCommandParameters parameters,
                                    @NonNull final BaseController controller,
                                    @SuppressWarnings(WarningType.rawtype_warning) @NonNull final CommandCallback callback,
                                    @NonNull final String publicApiId) {
         super(parameters, controller, callback, publicApiId);
+        checkAndRecordTaskInformation(parameters);
     }
 
     public InteractiveTokenCommand(@NonNull InteractiveTokenCommandParameters parameters,
                                    @NonNull List<BaseController> controllers,
                                    @SuppressWarnings(WarningType.rawtype_warning) @NonNull CommandCallback callback,
                                    @NonNull final String publicApiId) {
         super(parameters, controllers, callback, publicApiId);
+        checkAndRecordTaskInformation(parameters);
     }
 
+    private void checkAndRecordTaskInformation(InteractiveTokenCommandParameters parameters){
+        final String methodName = ":checkAndRecordTaskInformation";
+        final Context applicationContext = parameters.getAndroidApplicationContext();
+        final PackageManager packageManager = applicationContext.getPackageManager();
+        try {
+            ActivityInfo startActivityInfo = packageManager.getActivityInfo(parameters.getActivity().getComponentName(), 0);
```

Comment:
IIRC, OneAuth added some scenarios where the calling `Activity` might be `null`, we may want to be defensive here and check it's `!= null` before requesting the `componentName`

/cc @irving-as

Reply:
Not sure how that works... when would activity be null here?

Reply:
@irving-as Something you can comment on? As I recall, there is behavior where the broker / interactive flows will be invoked using only an `Activity` context